### ISSUE TITLE
Depending on # rows in grid - date picker doesn't display entire datepicker

### ISSUE THREAD

**Issue by PatriceBriggs** [Joiler/ui-grid-edit-datepicker#3]
I have this datepicker working in ui-grid.  Thank you!  There is an issue that I'm trying to resolve where the entire datepicker doesn't display if there are only a few rows in the grid.
![datepickerdisplayissue](https://cloud.githubusercontent.com/assets/9360864/13403160/8bd9d112-dec8-11e5-9a5a-bb588cbb3458.PNG)

**Comment by istvanskeri** [comment]
Yes the grid height has to be set to the height of the picker.

**Comment by Joiler** [comment]
Yes, thanks, I see this issue. I'm going to fix it this weekend.

**Comment by Joiler** [comment]
This issue has been fixed. I added **datepicker-append-to-body** property from [Angular UI Grid Datepicker](https://angular-ui.github.io/bootstrap/#/datepicker) so it should work correctly now. Plunker link where grid has little height: http://plnkr.co/edit/K64eD0COYBvwwZIq57wZ?p=preview
